students/dongjunKim/users/models.py

```diff
@@ -0,0 +1,16 @@
+from django.db import models
+
+class User(models.Model):
+    name     = models.CharField(max_length=30)
+    email    = models.CharField(max_length=100)
+    password = models.CharField(max_length=25)
+    phone    = models.CharField(max_length=15)
+    class Meta:
+        db_table = 'users'
+
+    #다음과 같은 필드들이 추가될 수 있음.
+    #주민번호, 성별, 주소, ..
+    
+    #향후 프론트 파트와의 협업을 진행할 때 달라질 수 있음.
+    #현재 진행하고 있는 프로젝트는 '위스타그램 - 벡앤드' 이므로
+    #기본 필드만 추가하였음.
```

Comment:
```suggestion
class User(models.Model):
    name     = models.CharField(max_length=30)
    email    = models.CharField(max_length=100)
    password = models.CharField(max_length=25)
    phone    = models.CharField(max_length=15)
    
    class Meta:
        db_table = 'users'
```
좋습니다 동준님! max_length는 조금 넉넉하게 잡아주시는 게 좋습니다. 
주석으로 고민의 흔적이 드러나는 부분 너무 좋습니다:+1: 하지만 머지되어야할 코드에서는 불필요하니 PR템플릿에 올려주시고 코드 내에서는 삭제해주세요!

Reply:
말씀해주신대로 필드 별 max_length 를 상향 조정하였습니다.